ssp

```diff
@@ -4397,6 +4398,9 @@ sub check_roots_cron_for_certain_commands {
         next if ( $checksum && $cron_ignore{$checksum} );
         if ( open my $cron_fh, '<', $cron ) {
             while (<$cron_fh>) {
+                if (m{ \A [^#]* (?:^|\s|\/)(backup|upcp)\s }x) {
```

Comment:
Should explicitly look for /usr/local/cpanel/bin/(backup|upcp) here to avoid any possible match with custom entries for third-party software (especially "backup" ones).

Reply:
Was about to say the same.  The only reason this could be more complicated was to assure the cron's **would actually fire** and not be disabled like:
```
SHELL=/bin/false
```
If that's there it bones the cron is may actually signal comped server.